### PR TITLE
Weaken highlighter priority

### PR DESCRIPTION
**Pull request by bolkhod**
# Description of changes made
Highlighter is now of "Additional Syntax" priority. This means that things like selecting code (with mouse), will be visible above the highlighter.

# Why is merge request needed
Selecting code with your mouse will now be fully visible. Before that the highlighter (for visualisation) obstructed everything

- [x] I have checked that I am merging into correct branch

### CODE REVIEW

**Review by martinmladenov — APPROVED** [review]

**Review by martinmladenov — CHANGES_REQUESTED** [review]
There's one more place where LAST is used (perhaps the opportunity can be used to get rid of some code duplication too :) )

**Comment by bolkhod** [comment]
> There's one more place where LAST is used (perhaps the opportunity can be used to get rid of some code duplication too :) )

Addressed both issues

**Review by martinmladenov — APPROVED** [review]

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/CoverageVisualisationService.kt [hunk 1]
```diff
@@ -39,22 +39,7 @@ class CoverageVisualisationService(private val project: Project) {
         val state = ApplicationManager.getApplication().getService(QuickAccessParametersService::class.java).state
         if (state.showCoverage) {
```
Comment (kirilvasilev16) [inline src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/CoverageVisualisationService.kt:40]: You can also remove this code duplication while you are at it? :)

Comment (bolkhod) [reply src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/CoverageVisualisationService.kt:40]: Oh wow, yeah even more can be reduced!

**Review by kirilvasilev16 — APPROVED** [review]
Nice catch of this bug!

**Review by martinmladenov — APPROVED** [review]

**Pull request merged**